libp2p/services/hpservice.nim

```diff
@@ -52,6 +51,9 @@ proc tryStartingDirectConn(self: HPService, switch: Switch, peerId: PeerId): Fut
   await sleepAsync(500.milliseconds) # wait for AddressBook to be populated
   for address in switch.peerStore[AddressBook][peerId]:
     try:
+      let isRelayed = address.contains(multiCodec("p2p-circuit"))
+      if isRelayed.isErr() or isRelayed.get():
+        continue
```

Comment:
Why not reuse `isRelayed()`?

Reply:
Do you mean the proc in relay.nim? If so, it is for a connection, this one is for a MA.